packages/lit-html-server/scripts/build.js

```diff
@@ -5,7 +5,7 @@ import path from 'node:path';
 await esbuild.build({
   bundle: true,
   entryPoints: ['./src/lit-html-server.js'],
-  external: ['lit-html', 'lit-html/*', './dom-shim.js', './element-renderer.js'],
+  external: ['@popeindustries/lit-html', './dom-shim.js', './element-renderer.js'],
```

Comment:
No need to use `*` here, as far as i read the [spec](https://esbuild.github.io/api/#external) it suffices without

>This means that --external:@foo/bar implicitly also means --external:@foo/bar/* which matches the import path @foo/bar/baz. So it marks all paths inside the @foo/bar package as external too.